src/platform.cpp

```diff
@@ -151,6 +151,78 @@ std::string Platform::FindStateCacheFolder() {
     return ".";
 }
 
+#elif defined(__APPLE__)
+#include <CoreFoundation/CoreFoundation.h>
+#include <pwd.h>
+#include <sys/param.h>
+#include <sys/stat.h>
+std::string Platform::FindResourceFolder() {
+    static std::string found_path;
+    static bool found = false;
+    if (found) return found_path;
+
+    CFURLRef resourceURL = CFBundleCopyResourcesDirectoryURL(CFBundleGetMainBundle());
+    char resourcePath[PATH_MAX];
+    if (CFURLGetFileSystemRepresentation(resourceURL, true, (UInt8 *)resourcePath, PATH_MAX)) {
+        if (resourceURL != NULL) {
+            CFRelease(resourceURL);
+        }
+        found_path = resourcePath;
+        found = true;
+        return found_path;
+    }
+
+    puts("cant find a resources folder, will try to load from cwd");
```

Comment:
ik puts is used elsewhere but now that we have spdlog all the (new) puts should look like `spdlog::get("discord")->warn`